.github/pull-request-template.md

```diff
@@ -0,0 +1,20 @@
+## Description
+
+Please include a summary of the change and which is is fixed.
```

Comment:
```suggestion
_Please include a summary of the change and which is is fixed._
```
Info text suggests only. let it be secondary.

Reply:
Actually I removed the string `Please ...` when I using this template. So adding underscore to the string might be decrease readability..